### DISCUSSION TITLE
add a little more explanation maybe?

### DISCUSSION THREAD

**Discussion by shug0131** [lcarcamo1526/TrainerRoad-to-Zwift#28] [Ideas]
Thanks for creating this tool.  I've been working out how to get the workout details _by hand_, so using the API for TR.  Your code is a guide for doing this, but I thought it might be helpful to others to explain how to do this. 

The basics are to log in to TR, and then look up in your browser a URL like

https://www.trainerroad.com/app/api/workoutdetails/20725

Here the tricky bit is to identify the number at the end. 

This can be done by going to calendar in TR, clicking on the workout you want, (ensure it is an indoor workout)  and then clicking in the top right image in the subwindow. This redirects to a url like

https://www.trainerroad.com/app/cycling/workouts/20725-bald  

and so the magic number you need is provided there. 

Clearly, what you do with the output from the API is whole load of work to process into whatever format you want.

Maybe you could put this explanation into a FAQ or other document within your github repo?

**Comment by lcarcamo1526** [comment]
Hey @shug0131 thanks for the suggestion, It has been a while since my last commit on this project hehe, but I'll open a FAQ, if you would like to modify it feel free to create a pull request :)
